src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1575,27 +1574,19 @@ private void AttachDebuggerToProcess(Process process)
         /// </summary>
         private static string MapReturnCodeToErrorMessage(int returnCode)
         {
-            string errorMessage = string.Empty;
-            switch (returnCode)
+            string errorMessage = returnCode switch
             {
-                case 2:
-                    errorMessage = ProcessResources.AttachDebuggerReturnCode2;
-                    break;
-                case 3:
-                    errorMessage = ProcessResources.AttachDebuggerReturnCode3;
-                    break;
-                case 8:
-                    errorMessage = ProcessResources.AttachDebuggerReturnCode8;
-                    break;
-                case 9:
-                    errorMessage = ProcessResources.AttachDebuggerReturnCode9;
-                    break;
-                case 21:
-                    errorMessage = ProcessResources.AttachDebuggerReturnCode21;
-                    break;
-                default:
-                    Diagnostics.Assert(false, "Unreachable code.");
-                    break;
+                2 => ProcessResources.AttachDebuggerReturnCode2,
+                3 => ProcessResources.AttachDebuggerReturnCode3,
+                8 => ProcessResources.AttachDebuggerReturnCode8,
+                9 => ProcessResources.AttachDebuggerReturnCode9,
+                21 => ProcessResources.AttachDebuggerReturnCode21,
+                _ => string.Empty
+            };
+
+            if (string.IsNullOrEmpty(errorMessage))
+            {
+                Diagnostics.Assert(false, "Unreachable code.");
```

Comment:
Eh, now the comment is not correct. And please replace `false` with real condition and remove `if`.

Reply:
@iSazonov Will replace with `Diagnostics.Assert(!string.IsNullOrEmpty(errorMessage), "Unreachable code.");`.

Should we also change the message to something else like `error message should not be null or empty`?

Reply:
Looks good.